interface/super/edit_list.php

```diff
@@ -1229,26 +1318,19 @@ function mysubmit() {
             }
         } elseif ($list_id == 'issue_types') {
             $res = sqlStatement("SELECT * FROM issue_types " .
-                "ORDER BY category, ordering ASC");
+                "ORDER BY category, ordering ".$sql_limits);
             while ($row = sqlFetchArray($res)) {
                 writeITLine($row);
             }
             for ($i = 0; $i < 3; ++$i) {
                 writeITLine(array());
             }
         } else {
-            /*
-             *  Add edit options to show or hide in list management
-             *   If the edit_options setting of the main list entry is set to 0,
-             *    then none of the list items will show.
-             *   If the edit_options setting of the main list entry is set to 1,
-             *    then the list items with edit_options set to 1 will show.
-             */
             $res = sqlStatement("SELECT lo.*
-                         FROM list_options as lo
-                         right join list_options as lo2 on lo2.option_id = lo.list_id AND lo2.list_id = 'lists' AND lo2.edit_options = 1
+                         FROM list_options AS lo
+                         RIGHT JOIN list_options AS lo2 ON lo2.option_id = lo.list_id AND lo2.edit_options = 1
```

Comment:
looks like the change above is reverting a recent bug fix

Reply:
Whats fix?